app/app/urls.py

```diff
@@ -84,7 +84,11 @@
     # personal_tokens
     re_path(r'^token/quickstart/?', ptokens.views.quickstart, name='ptoken_quickstart'),
     re_path(r'^token/faq/?', ptokens.views.faq, name='ptokens_faq'),
-    path('tokens/<str:token_state>/', ptokens.views.tokens, name='token'),
+    path('tokens/redemptions/<int:redemptionId>/', ptokens.views.ptoken_redemption, name='token_redemption'),
```

Comment:
are we expecting the path to be _tokens_ or _ptokens_ here?

Reply:
to have consitency, i'll rename to ptokens